docs/source/usage/index.rst

```diff
@@ -225,6 +225,58 @@ In order to use an external provisioning service, simply attach the node to the
 | Attach Network to Node        | ``openstack esi node network attach (--network <network> | --port <port>) <node>`` |
 +-------------------------------+------------------------------------------------------------------------------------+
 
+Serial Console
+---------------
+
+Ironic nodes can be accessed through configuring a web or serial interface.
+This documentation was taken from the Ironic Xena release.
```

Comment:
I don't think there's a need for "This documentation was taken from..." anymore.